addons/zebra-striping/addon.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "Alternating nested block colors",
+  "description": "Makes blocks of the same category alternate between lighter and darker shades when nested inside each other.",
+  "credits": [
+    {
+      "name": "CST1229",
+      "link": "https://github.com/CST1229"
```

Comment:
Just so you're aware, in TurboWarp will be replaced with a link to your Scratch profile instead